ivy/functional/backends/tensorflow/experimental/losses.py

```diff
@@ -0,0 +1,56 @@
+import tensorflow as tf
+from tensorflow import nn
+from typing import Optional, Tuple
+
+from functools import reduce
+
+import ivy
+
+from ivy.functional.ivy.experimental.linear_algebra import _check_valid_dimension_size
+
+from ivy.func_wrapper import with_unsupported_dtypes, with_supported_dtypes
+from .. import backend_version
+
+
+@with_unsupported_dtypes(
+    {"2.9.1 and below": ("int", "float16", "bfloat16")}, backend_version
+)
+
+def ctc_loss(
+    log_probs: tf.Tensor,
+    targets: tf.Tensor,
+    input_lengths: tf.Tensor,
+    target_lengths: tf.Tensor,
+    /,
+    *,
+    blank: int = 0,
+    reduction: str = "mean",
+    axis: int = 1,
+    epsilon: float = 1e-08,
+    out: Optional[tf.Tensor] = None,
+) -> tf.Tensor:
+
+    log_probs = tf.transpose(log_probs, perm=[axis, 0, 1])
+    #Compute the CTC loss using the Pytorch implementation
+    ctc_loss = nn.ctc_loss(
```

Comment:
in a backend implementation, you should only use functions of that backend only, also there is no such function